### DISCUSSION TITLE
Version help

### DISCUSSION THREAD

**Discussion by Computer-help** [WinMerge/winmerge#1557] [Q&A]
I want to compare files in Windows 10 and not sure which version of Winmerge to pick. What is the difference between the choices that come up when I type in Winmerge in the search box (download for windows 10 64 bit) (portable) (tool) (64)?

**Comment by sdottaka** [comment]
Are you about to download and install WinMerge?

In most cases, Windows 10 is a 64bit version, so please download and install the following file.

https://github.com/WinMerge/winmerge/releases/download/v2.16.24/WinMerge-2.16.24-x64-Setup.exe

If you are not an administrator on your PC, please download and install the file below.

https://github.com/WinMerge/winmerge/releases/download/v2.16.24/WinMerge-2.16.24-x64-PerUser-Setup.exe

Alternatively, if you want to run WinMerge without installing WinMerge, download the ZIP file below, expand the ZIP file, and execute the expanded WinMergeU.exe.

https://github.com/WinMerge/winmerge/releases/download/v2.16.24/winmerge-2.16.24-x64-exe.zip

**Comment by Computer-help** [comment]
Hello,

Thank you for the reply. My laptop (which I'm administrator for) comes up
with several options when WinMerge is typed into the search box. i was just
trying to find out if the differences between the choices the search box
gave me.

Thanks, Rob

**Comment by sdottaka** [comment]
By search box do you mean the one in the image below?

If so, click on the one labeled App to launch WinMerge.

I think others (download for windows 10 64 bit, etc.) are just web search results.
![image](https://user-images.githubusercontent.com/98126/199849347-b8cbf638-208c-46a5-916f-0656be5b3d69.png)
